### ISSUE TITLE
Interface inventory is incorrectly configured

### ISSUE THREAD

**Issue by calvissuperman** [PrototypeTrousers/Applied-Energistics-2#176]
**Describe the bug**

 The ME Interface acts as a storage block as well as a network block. The problem with this is when you use the ME Interface as an import bus. When items are piped into the ME Interface after a crafting operation is completed, there is a small window of opportunity for the machine it's connected to to pull out the items you just pumped in. Example of this is very apparent with the Ender IO mod.

**To Reproduce**

Place a ME Interface directly behind an Alloy Furnace from enderIO, and set the alloy furnace to push/pull from that side. Now setup a recipe that creates an item that can be used in further alloy recipes, and every so often when a crafting operation is completed the alloy furnace will push the item into the ME Interface, but pull it back out immediately and place the completed item up in the input slots of the Alloy furnace.

This is an example, the behavior mentioned above happens with many other mods as well. Not EnderIO specific.

**Expected behavior**

I expect it to not pull those items out immediately, and let the ME Interface import them back into the ME system.

**Additional context**

To me the answer to this issue is complicated, as the machine is actively pulling items from the ME Interface. When set to not pull, the interface will not push items into the machine. So it's a mismatch of protocol between this mod and others on how input/output faces interact with the ME Interface. There needs to be a way for machines to push items into an ME Interface, without the item showing up in the ME Interface inventory to be allowed to be pulled back out by that same machine immediately.

**Environment**

- Minecraft Version: 1.12.2
- AE2 Version: 0.52.11
- Forge Version:

**Comment by PrototypeTrousers** [comment]
AE item movement happens on its own tick handler, and does not happen immediately after an item is placed on an interface, instead it's notified that it has work to do and schedules itself with the tickhandler.

TL;DR expected behaviour.

I suggest a dedicated interface to collect.

**Comment by calvissuperman** [comment]
Thank you for the quick reply! Wish I could get away with less blocks, but sometimes that's just how it is hahah.

**Comment by IntegerLimit** [comment]
You could disable the auto pull from the interface, as the interface auto-pushes into inventories next to it for crafts. Try it out, I know it works with machines from other mods, but I am unsure if it works with EnderIO machines.
